drafts/bitcartel-payment-disclosure/draft.rst

```diff
@@ -0,0 +1,339 @@
+::
+
+  ZIP: 303
+  Title: Payment Disclosure
+  Owners: Deirdre Connolly <deirdre@zfnd.org>
+  Original-Author: Simon Liu
+  Status: Implemented (zcashd)
+  Category: Informational
+  Created: 2017-02-22
+  License: MIT
+
+
+Abstract
+--------
+
+This ZIP describes a method of proving that a payment was sent to a shielded address.  In the typical case, this means enabling a sender to present a proof that they transferred funds to a recipient's shielded address.  The method described will be compatible with the Zcash Protocol [#protocol]_ deployed at the launch of the Zcash network. See zcash issues `2036 <https://github.com/zcash/zcash/issues/2036>`_ and `737 <https://github.com/zcash/zcash/issues/737>`_ for context.
+
+
+Motivation
```

Comment:
Another motivation is to simply share visibility of a shielded transaction with a third party.